zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/RestoreFromBackupTool.java

```diff
@@ -290,9 +299,14 @@ public boolean runWithRetries(CommandLine cl) {
         return true;
       } catch (IllegalArgumentException re) {
         System.err.println(
-            "Restore attempt failed due to insufficient backup files; attempting again. " + tries
-                + "/" + MAX_RETRIES + ". Error message: " + re.getMessage());
-        break;
+            "Restore attempt failed due to insufficient backup files. Error message: "
```

Comment:
insufficient backup files -> this is a little ambiguous - what does insufficient mean in this case? could we reword this message?